src/lib/lazy-load.js

```diff
@@ -46,19 +46,21 @@ const measure = (uid) => {
 	perfMeasure(uid, `START:${uid}`, `END:${uid}`);
 };
 
-// const report = (name) => {
-// 	const entry = performance.getEntriesByName(name)[0];
-// 	const selector = `img[data-uid="${name}"]`;
-// 	const img = document.querySelector(selector);
-// 	if(img && img.currentSrc){
-// 		const eventData = {
-// 			category: 'lazy-image-load',
-// 			action: 'timing',
-// 			data: {src:img.currentSrc, duration:entry.duration}
-// 		};
-// 		broadcast('oTracking.event', eventData);
-// 	}
-// };
+const report = (name) => {
+	if (performance.getEntriesByName) {
```

Comment:
could move https://github.com/Financial-Times/n-image/blob/41053423f5e2529d3a58a471e2b354a5d085eaf6/src/lib/lazy-load.js#L9 into a shared `const`, and this can be `if (performance && performance.getEntriesByName) {`

Means we'll get reporting in more than just Chrome